operator/pkg/gateway-api/tlsroute.go

```diff
@@ -102,6 +102,11 @@ func (r *tlsRouteReconciler) enqueueRequestForBackendService() handler.EventHand
 	return handler.EnqueueRequestsFromMapFunc(r.enqueueFromIndex(backendServiceIndex))
 }
 
+// enqueueRequestForRequestGrant makes sure that TLS Routes in the same namespace are reconciled
```

Comment:
This comment feels outdated?

Reply:
regarding the part about `in the same namespace` ? -> `all TLS Routes are reconciled`

Reply:
renamed function and adapted comment